src/test/kotlin/uk/gov/dluhc/emsintegrationapi/config/QueueConfigurationTest.kt

```diff
@@ -16,8 +15,7 @@ internal class QueueConfigurationTest {
             PROXY_APPLICATION_QUEUE.name,
             POSTAL_APPLICATION_QUEUE.name,
             DELETED_PROXY_APPLICATION_QUEUE.name,
-            DELETED_POSTAL_APPLICATION_QUEUE.name,
-            EMS_APPLICATION_PROCESSED_QUEUE.name,
```

Comment:
I think this test is silly, but it looks like we're unexpectedly removing things to do with `EMS_APPLICATION_PROCESSED_QUEUE`. I think we're needing to be cautious and check through if we're accidentally deleting anything we shouldn't be.